man/scrot.txt

```diff
@@ -68,8 +67,11 @@ OPTIONS
   -u, --focused             Use the currently focused window.
   -v, --version             Output version information and exit.
   -z, --silent              Prevent beeping.
-  -                         Redirection to standard output. The output image
-                            format is PNG.
+  FILE | -F, --file FILE    Specify the output file. A "-" output file outputs
```

Comment:
Same as above, I'd revert this and just put in the description that -F and FILE are mutually exclusive.

Reply:
Maybe it would be good to imitate st's manual here too? st's manual only has the equivalent of the `-F, --file FILE` bit, and the text beside it explains that a non-option argument works as if it was the option.

Reply:
That seem fine. Would need to explain that `-` acts as stdout in the text also.